tools/msconvert/test-data/small-peakpicking-vendor-allMS.mzML

```diff
@@ -79,15 +84,15 @@
     </dataProcessingList>
     <run id="small" defaultInstrumentConfigurationRef="IC1" startTimeStamp="2005-07-20T14:44:22Z" defaultSourceFileRef="RAW1">
       <spectrumList count="48" defaultDataProcessingRef="pwiz_Reader_Thermo_conversion">
-        <spectrum index="0" id="controllerType=0 controllerNumber=1 scan=1" defaultArrayLength="1750">
+        <spectrum index="0" id="controllerType=0 controllerNumber=1 scan=1" defaultArrayLength="1810">
```

Comment:
For future reference, this is changing because pwiz is now including exception/reference peaks from the Thermo reader.